superset/views/core.py

```diff
@@ -1128,18 +1129,32 @@ def explorev2(self, datasource_type, datasource_id):
 
     @log_this
     @has_access
-    @expose('/explore/<datasource_type>/<datasource_id>/')
+    @expose('/explore/<datasource_type>/<datasource_id>/', methods=['GET', 'POST'])
```

Comment:
Is there a reason we have these trailing `/` everywhere? Can we (start) to get rid of them?
e.g. Add another line
`@expose('/explore/<datasource_type>/<datasource_id>', methods=['GET', 'POST'])`

Reply:
I asked Max about this, he feel this is just flask url style. But because of backward compatibility, I can't remove the trailing `/`, otherwise users' saved explore url won't work.

Reply:
Trailing slashes are pretty common in the Django / Python world.